### ISSUE TITLE
Consider generating GitHub Pages to show what data is available in this repo

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-data-oasis#2]

**Comment by ronaldtse** [comment]
@andrew2net : @CAMOBAP can help apply this workflow to all relaton-data-* repositories.

**Comment by andrew2net** [comment]
@ronaldtse the most work here is to create indexes in all relaton-data-* repositories. Not every repository has an index yet. And I started thinking about relaton-index gem. The reasons are:
- The existing indexes have different structures. I think we should unificate them.
- Some parts of code repeated in index generators.

The index gem should:
- register fields for indexing
- read existed index file or create a new emplty index if file odesn't exist
- add element to index
- search element in index
- save index to file

As we have a same formated index in every relato-data-* repository, we can easely generate index pages using same Jekyll template.
Any thoughts?

**Comment by ronaldtse** [comment]
@andrew2net I agree that the very first thing is to have the same index structures across all Relaton repositories. 

An index is really a subset of the Relaton XML/YAML information of a Relaton collection, in a form that can be easily accessed and searched.

Remember that the Metanorma site generate command creates the mini site based on RXL (Relaton XML) information? This is exactly like the purpose of the Relaton index site -- look at standards.calconnect.org. 

We should extract that code and consolidate such code into relaton-index.
